tests/logger_logging/test_local_decorating.py

```diff
@@ -55,7 +85,18 @@ def test_local_log_decoration_inside_transaction(logger):
     @background_task()
     def test():
         exercise_logging(logger)
-        assert logger.caplog.records[0] == get_metadata_string('C', True)
+        assert logger.caplog.records[0] == get_metadata_string("C", True)
+
+    test()
+
+
+@reset_core_stats_engine()
+def test_local_log_decoration_inside_transaction_with_json(logger):
```

Comment:
Do we need another test where it is outside the transaction?

Reply:
I initially thought it might be redundant but in the grand scheme of all the tests we have, adding this one is a drop in the ocean.  Done!